stacks-common/src/util/mod.rs

```diff
@@ -66,8 +66,8 @@ pub enum HexError {
 impl fmt::Display for HexError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
-            HexError::BadLength(n) => write!(f, "bad length {} for sha256d hex string", n),
-            HexError::BadCharacter(c) => write!(f, "bad character {} in sha256d hex string", c),
```

Comment:
made more sense when this said "in hex string"

Reply:
That's weird, my code had the opposite delta.  Will revert.

Reply:
Is this a change you want to revert still?